default.build

```diff
@@ -36,11 +36,11 @@
     <call target="build_windowsphone81" />
     <call target="build_windows10" />
     <call target="build_psm" />
-    <call target="build_web" />
```

Comment:
JSIL fails to compile, build server already has it compiled so it never does it again.